run/john.zsh_completion

```diff
@@ -48,19 +48,19 @@ _arguments -C -s \
   '(--no-mask)'--no-mask'[used with --test for alternate benchmark w/o mask]' \
   '(--skip-self-tests)'--skip-self-tests'[skip self tests]' \
   '(--test-full)'--test-full'[run more thorough self-tests]' \
-  '(--users'--users='[do not load these users only]:user login or uid' \
-  '(--groups'--groups='[do not load these group only]:group gid' \
-  '(--salts'--salts='[load salts with(out) COUNT (to MAX) hashes]:salts' \
-  '(--costs'--costs='[load salts with(out) cost value Cn (to Mn)]:costs' \
+  '(--users)'--users=-'[do not load these users only]:user login or uid' \
+  '(--groups)'--groups=-'[do not load these group only]:group gid' \
+  '(--salts)'--salts=-'[load salts with(out) COUNT (to MAX) hashes]:salts' \
+  '(--costs)'--costs=-'[load salts with(out) cost value Cn (to Mn)]:costs' \
   '(--make-charset)'--make-charset=-'[make a charset file. It will be overwritten]:charset file:_files' \
   '(--pot)'--pot=-'[pot file to use]:pot file:_files' \
-  '(--node)'--node=-'[this node's number range out of TOTAL count]:number' \
+  '(--node)'--node=-'[this nodes number range out of TOTAL count]:number' \
   '(--subformat)'--subformat=-'[pick a benchmark format for --format=crypt]:format to use:->list-formats' \
   '(--mkpc)'--mkpc=-'[request a lower max. keys per crypt]:number' \
   '(--min-length)'--min-length=-'[request a minimum candidate length in bytes]:number' \
   '(--max-length)'--max-length=-'[request a maximum candidate length in bytes]:number' \
   '(--length)'--length=-'[shortcut for --min-len=N --max-len=N]:number' \
-  '(--field-separator-char)'--field-separator-chat=-'[use "C" instead of the "":" in input and pot files]:char' \
+  '(--field-separator-char)'--field-separator-chat=-'[use "C" instead in input and pot files]:char' \
```

Comment:
There's a typo in the original not yet fixed here: "chat" in place of "char". And the edited description is now weird, perhaps simply replace the `"":"` with `colon` instead.

Reply:
put ":" back in

Reply:
Thanks. Will you fix the "chat" typo as well, please?

And, can you please force-push all of this as just one commit, instead of adding commits? Alternatively, please let us know if we should use GitHub's "squash and merge" - dirtier, but works. It is pointless to keep this PR's individual commits in our bleeding-jumbo branch's history as they're only part of this PR's short-lived history.

Reply:
done